Wireframe/style.css

```diff
@@ -101,3 +104,18 @@ article {
     grid-column: span 3;
   }
 }
+@media screen and (min-width:320px) and (max-width:640px) {
+
+  body {
+      max-width: 100%;
+      overflow-x: hidden;
+    }
+main {
+  display: block; 
+  }
```

Comment:
well done, your website is now also mobile friendly. Remember though there is this convention while defining breakpoints in your queries. 
https://www.browserstack.com/guide/responsive-design-breakpoints you can check this article. Generally speaking, these break points are used by majority of developer.